### DISCUSSION TITLE
Is the docker container still updated?

### DISCUSSION THREAD

**Discussion by prodigy** [SiaFoundation/siad#166] [Q&A] [answered]
Hi there,

There have been a few releases here, while the latest container version is from about 6 months ago (https://github.com/SiaFoundation/siad/pkgs/container/siad).

Is this still the up-to-date addres, as is documented in this very repo (image `ghcr.io/siafoundation/hostd`)?
If yes, when will the docker image be updated?
If not, is there a new one?

Thank you for your effort.

**Comment by n8mgr** [comment, marked as answer]
`siad` and the `siad` package are deprecated. It will not be updated. 

- If you want to provide storage, use https://github.com/SiaFoundation/hostd (`ghcr.io/siafoundation/hostd:latest`)
- If you want to renter storage, use https://github.com/SiaFoundation/renterd (`ghcr.io/siafoundation/renterd:latest`)
